convention-develocity-gradle-plugin/plugins/gradle-5-or-newer/src/main/java/com/myorg/ConventionDevelocityGradlePlugin.java

```diff
@@ -43,24 +41,24 @@ public void apply(Object target) {
     }
 
     private void configureGradle6OrNewer(Settings settings) {
-        settings.getPluginManager().apply(GradleEnterprisePlugin.class);
+        settings.getPluginManager().apply(DevelocityPlugin.class);
         settings.getPluginManager().apply(CommonCustomUserDataGradlePlugin.class);
-        configureGradleEnterprise(settings.getExtensions().getByType(GradleEnterpriseExtension.class), settings.getGradle().getStartParameter());
-        configureBuildCache(settings.getBuildCache(), settings.getExtensions().getByType(GradleEnterpriseExtension.class));
+        configureDevelocity(settings.getExtensions().getByType(DevelocityConfiguration.class), settings.getGradle().getStartParameter());
+        configureBuildCache(settings.getBuildCache(), settings.getExtensions().getByType(DevelocityConfiguration.class));
     }
 
     private void configureGradle5(Project project) {
-        project.getPluginManager().apply(BuildScanPlugin.class);
+        project.getPluginManager().apply(DevelocityPlugin.class);
```

Comment:
Our docs say the Develocity Gradle plugin is compatible with Gradle 5.x and higher as opposed to having to use the Build Scan plugin. I tested it using the sample projects and it works.

https://docs.gradle.com/develocity/gradle-plugin/current/#applying_the_plugin